cms/static/js/certificates/views/signatory_editor.js

```diff
@@ -2,18 +2,27 @@
  * This class defines an editing view for course certificates.
  * It is expected to be backed by a Certificate model.
  */
-define(['js/views/list_item_editor', 'js/certificates/models/signatory' ,"js/utils/templates", 'underscore', 'jquery', 'gettext'],
-function(ListItemEditorView, Signatory, TemplateUtils, _, $, gettext) {
+define(['js/views/list_item_editor',"js/utils/templates", 'underscore', 'jquery', 'gettext'],
```

Comment:
@mattdrayer  we have the signatory-editor.underscore file. I think in my Last PR i have added this file. Let me know if anything i am missing at my end ?